raiden/api/python.py

```diff
@@ -1728,11 +1801,11 @@ def create_light_client_payment(
                 locked_transfer.message_identifier,
                 locked_transfer,
                 False,
-                payment.payment_id,
                 creator_address,
                 order,
                 LightClientProtocolMessageType.PaymentSuccessful,
-                self.raiden.wal
+                self.raiden.wal,
+                payment.payment_id
```

Comment:
i dont understand the addition of payment.payment_id parameter and how is this related with settlement.

Reply:
It's not, but we forgot to do that in the refactor of the database. Basically we had the payment id defined as an integer in the middle and when you put None for those cases where you have to do that since is not a payment related message, it shows you a warning, so to solve this you need to put payment_id as a default parameter = None, but to do that you need to move it to the end. I can remove this change if it's a problem, i just wanted to keep the code clean that's all.

Reply:
It makes sense to me.